src/sysstr.h

```diff
@@ -35,6 +35,16 @@
 #define IsDigit(ch) (isdigit((unsigned int)ch))
 
 
+/****************************************************************************
+**
+*F  IsIdent( <ch> )
+*/
```

Comment:
Seeing that all other functions in here have (at least a short) explanation on what they do, maybe you could add one here?

Reply:
This should be added at some point, but since I just moved this code around, and since two other PRs depend on this PR, I'd rather do this in a separate PR, and merge this PR here right away.
Same for the comment at the file start -- note that these are either missing or wrong for basically all `sys*.{c,h}` files at this point...